app.rb

```diff
@@ -12,6 +12,7 @@
 require 'everypolitician/popolo'
 
 require_relative './lib/popolo_helper'
+require_relative './helpers/html'
```

Comment:
I thought about that and I think it may be a good idea. I've seen that in other frameworks as Rails or Middleman and I think it helps to keep concerns separated. But I am happy to put this file under lib if we prefer it so.